frontend/src/workspace/funbar/FunBar.js

```diff
@@ -127,8 +127,8 @@ function FunBar(props) {
 
     return (
       <Group // Image Button on blue bar
-        x={funBarDimensions.imagePlaceX}
-        y={height-nodeDimensions.renderSideLength-535}
+      x={width*.82}
+      y={width/-8.6} //Doesn't work properly for window size changing
```

Comment:
Shouldn't this be fixed before we merge?

Reply:
Also I can't put a comment for some reason but on line 156, it sound be "its" instead of "it's"